topologies/kne/nokia/srlinux/dutate.textproto

```diff
@@ -14,6 +14,34 @@ nodes: {
             }
         }
     }
+    services: {
```

Comment:
can you add this to the other topos too?

Reply:
Not a problem, will push a commit in a few